pywbem_mock/_wbemconnection_mock.py

```diff
@@ -1421,7 +1425,7 @@ def _filter_properties(obj, property_list):
         Remove properties from an instance or class that aren't in the
         plist parameter
 
-        obj(:class:`~pywbem.CIMClassName` or :class:`~pywbem.CIMClassName):
+        obj(:class:`~pywbem.CIMClass` or :class:`~pywbem.CIMClass):
             The class or instance from which properties are to be filtered
```

Comment:
I assume this should have been CIMClass or CIMInstance?

Reply:
Yes. FIXED.